sound/soc/sof/pcm.c

```diff
@@ -460,6 +464,7 @@ static int sof_pcm_close(struct snd_soc_component *component,
 {
 	struct snd_soc_pcm_runtime *rtd = asoc_substream_to_rtd(substream);
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(component);
+	const struct sof_ipc_pcm_ops *pcm_ops = sdev->ipc->ops->pcm;
```

Comment:
This should go to the patch where you introduce the new "open" ops method.

Reply:
sure, thanks